### PR TITLE
Feature/7 relations tab

### PR DESCRIPTION
**Pull request by Moltenship**
#7

### CODE REVIEW

File: src/features/VisualNovel/components/RelationCard/RelationCard.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    minW="72"
+    overflow="hidden"
+    bg="gray.100"
+    gridTemplateRows="100%"
+  >
+    <Image src={image ?? undefined} fallbackSrc={vnPosterPlaceholder} borderRadius="lg" h="full" w="full" objectFit="cover" />
+    <GridItem p="3">
+      <Flex direction="column" justify="space-between" h="full">
+        <Box>
+          {!isOfficial && <Text as="span" fontSize="xs">Unofficial</Text>}
```
Comment (NightOfNights) [inline src/features/VisualNovel/components/RelationCard/RelationCard.tsx]: You can avoid `!` inversion with `isOfficial === false`

File: src/api/services/visualNovelRelationService.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    [RelationType.Prequel]: 'Prequel',
+    [RelationType.SameSeries]: 'Same series',
+    [RelationType.ParentStory]: 'Parent story',
+  };
+
+  /**
+   * Gets relation type for related novel.
+   * @param relatedNovelId Related novel id.
+   */
+  export const getRelationData = (relatedNovelId: number, novel: VisualNovel): RelationData => {
```
Comment (NightOfNights) [inline src/api/services/visualNovelRelationService.ts]: Use `VisualNovel['id']` type instead

File: src/api/mappers/visualNovelMapper.ts [hunk 3]
```diff
@@ -60,7 +61,7 @@ const visualNovelRelatedFromDto = (dto: VisualNovelRelatedDto): VisualNovelRelat
   id: dto.id,
   isOfficial: dto.official,
   originalName: dto.original,
-  relation: dto.relation,
+  relation: dto.relation as RelationType,
```
Comment (NightOfNights) [inline src/api/mappers/visualNovelMapper.ts]: ?

File: src/api/services/visualNovelService.ts [hunk 4]
```diff
@@ -16,3 +16,16 @@ export const fetchFullVisualNovel = async(id: string): Promise<VisualNovel> => {
 
   return data.data.items.map(dto => visualNovelFromDto(dto))[0];
 };
+
+/**
+ * Fetches visual novels by vnIds.
+ * @param ids Array of vn ids.
+ */
+export const fetchVisualNovelByIds = async(ids: number[]): Promise<VisualNovel[]> => {
+  const { data } = await http.post<DataWrapper<VisualNovelDto>>(
```
Comment (NightOfNights) [inline src/api/services/visualNovelService.ts]: `ids: VisualNovel['id'][]`

File: src/features/VisualNovel/components/VisualNovelTabs/VisualNovelTabs.tsx [hunk 5]
```diff
@@ -33,6 +33,10 @@ export const VisualNovelTabs: VFC<Props> = memo(({ id }) => {
   const location = useLocation();
   const [tabIndex, setTabIndex] = useState(getInitialTabIndex(location.pathname));
```
Comment (NightOfNights) [inline src/features/VisualNovel/components/VisualNovelTabs/VisualNovelTabs.tsx]:
```suggestion
  const [tabIndex, setTabIndex] = useState(FIRST_INDEX);
```

**Review by moetazaneta — APPROVED** [review]

**Review by NightOfNights — APPROVED** [review]

**Pull request merged**